README.md

```diff
@@ -61,80 +58,104 @@ Install NPM dependencies
 
 npm i
 ```
+
 ---
 
+## Environment Set Up
 
+### Environment Variables
 
-# Environment Set Up
+To run this application locally, the first step is to create a local .env file and set the environment variables with the appropriate values. You can duplicate .env.example file and rename it to .env
 
+```bash
 
-## Set up Database
+# src/common/.env.example
 
-### Start Postgres (If not using the Postgres.app)
+PORT=3000
+BASE_URL=http://localhost:3000
```

Comment:
Why does it already have a port as part of the url? Is the above port a separate setting?